OpenUpSA/wazimap-ng
# Make indicator and geography unique together to fix issue of duplicate Indicator data

## Description
Issue:
There were duplicate indicator data objects if the user ran task more than once in quick intervals
Fix:
An immediate fix would be to add unique together on geography and indicator so we can not have duplicate indicator data objects and if there is one task will throw an error and will be shown as failed task

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-327

## How to test it locally
Without running the migration open a variable and click save multiple times.
Check for Indicator data count after all task are done processing.
You will see that Indicator data objects are equal to number of geos * the number of time user saved indicator.

Now run migration and try to do this again
In this case user will get exception for one task and one task will return successful message

## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Commits


### Code Quality


### Testing
